datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mlmodel/mappers/MlModelMapper.java

```diff
@@ -0,0 +1,96 @@
+package com.linkedin.datahub.graphql.types.mlmodel.mappers;
+
+import java.util.stream.Collectors;
+
+import javax.annotation.Nonnull;
+
+import com.linkedin.data.template.RecordTemplate;
+import com.linkedin.datahub.graphql.generated.EntityType;
+import com.linkedin.datahub.graphql.generated.FabricType;
+import com.linkedin.datahub.graphql.generated.MLModel;
+import com.linkedin.datahub.graphql.generated.SourceCode;
+import com.linkedin.datahub.graphql.types.common.mappers.CostMapper;
+import com.linkedin.datahub.graphql.types.common.mappers.DeprecationMapper;
+import com.linkedin.datahub.graphql.types.common.mappers.InstitutionalMemoryMapper;
+import com.linkedin.datahub.graphql.types.common.mappers.OwnershipMapper;
+import com.linkedin.datahub.graphql.types.common.mappers.StatusMapper;
+import com.linkedin.datahub.graphql.types.mappers.ModelMapper;
+
+/**
+ * Maps Pegasus {@link RecordTemplate} objects to objects conforming to the GQL schema.
+ *
+ */
+public class MlModelMapper implements ModelMapper<com.linkedin.ml.MLModel, MLModel> {
```

Comment:
naming nit: Let's name "MLModelMapper" to keep the capitalization consistent across model & code